erizo/src/erizo/WebRtcConnection.cpp

```diff
@@ -29,6 +29,7 @@
 #include "rtp/LayerDetectorHandler.h"
 #include "rtp/QualityFilterHandler.h"
 #include "rtp/QualityManager.h"
+#include "rtp/PLIPacerHandler.h"
```

Comment:
I've been using camelcase also for abbreviations lately (NackGenerator). I don't have an strong opinion on what's better but we should be consistent